lib/second_base/databases.rake

```diff
@@ -113,7 +113,7 @@ end
   test:purge test:load_schema test:load_structure test:prepare
 }.each do |name|
   task = Rake::Task["db:#{name}"] rescue nil
-  next unless task
+  next unless task && SecondBase::Railtie.run_with_db_rake_tasks?
```

Comment:
`SecondBase::Railtie.run_with_db_rake_tasks?` needs to be mirrored in the rails 4 and rails 5 tasks too.